seed/seedwriter/writer.go

```diff
@@ -105,6 +128,16 @@ type SeedSnap struct {
 	local      bool
 	modelSnap  *asserts.ModelSnap
 	optionSnap *OptionsSnap
+
+	Components []SeedComponent
```

Comment:
After reading the code some more, I see why this is you used `SetInfo` to initialize this field, since there is already this deferred initialization of parts of this type. Maybe a comment here on this field, and move this next to the `Info` field that is in the same boat.

Reply:
I've moved and added a comment.